### ISSUE TITLE
Several CI issues after #61

### ISSUE THREAD

**Issue by ChenxingLi** [0glabs/0g-storage-node#62]
1. The CI workflow requries `sudo` when downloading `evmosd` in each single test, even if the file owners in `tests/tmp` are not root. 
2. After the first failure because of this issue, when I rerun the test script, `test_all.py` reports 
```
Traceback (most recent call last):
  File "/home/chenxing/vscode-workspace/0g-storage-node/tests/test_all.py", line 8, in <module>
    run_all(
  File "/home/chenxing/vscode-workspace/0g-storage-node/tests/utility/run_all.py", line 64, in run_all
    build_bsc(tmp_dir)
  File "/home/chenxing/vscode-workspace/0g-storage-node/tests/utility/build_binary.py", line 50, in build_bsc
    assert downloaded, "Cannot download download binary from github [%s]" % BSC_BINARY
AssertionError: Cannot download download binary from github [get]
```

**Comment by ChenxingLi** [comment]
The first issue is resolved after manually running `sudo snap install jq`. 

But the second issue persists.

**Comment by ChenxingLi** [comment]
Resolved by #63
